monero-outreach-docs/translations/pt/PSA/files/teamwork/teamwork_pt.md

```diff
@@ -0,0 +1,13 @@
+# Participação Comunitária
+
+O Monero é um projeto de código aberto completamente descentralizado. Qualquer pessoa com uma boa ideia e alguma iniciativa pode criar e liderar um projeto de apoio ao Monero. Se você não é um programador e está interessado em dar o seu apoio ao Monero de outras maneiras, existem muitas opções. Os projetos do Monero organizam-se em vários grupos de trabalho. Há um grupo de trabalho de idiomas focado em traduzir todo o conteúdo Monero para ajudar a estimular a adoção em todo o mundo, um grupo de trabalho focado na criação de ferramentas para facilitar melhores relações públicas e maior adoção do Monero, e até mesmo um grupo de trabalho focado no desenvolvimento de uma carteira de hardware para o Monero. Existem muitos outros grupos de trabalho que precisam da sua ajuda.
+
+Se você é um programador, pode se juntar a algumas centenas de outros profissionais que contribuíram para o código do Monero, através do GitHub.
+
+The first stop when it comes to community workgroup participation is the Monero Community Workgroup, which focuses on organizing all of the otheO primeiro local a visitar quando se trata de participação em grupos de trabalho da comunidade, é o Monero Community Workgroup, cujo foco é a organização de todos os restantes grupos de trabalho. Eles poderão dizer-lhe se a sua ideia já está implementada e com que pessoas trabalhar para levá-la ainda mais longe. Se a sua ideia for nova, eles irão direcioná-lo para alguns recursos para o ajudar transformar a sua ideia em realidade.
```

Comment:
Delete the English sentence:

"The first stop when it comes to community workgroup participation is the Monero Community Workgroup, which focuses on organizing all of the othe"